types/queries/insert_all_chapters.sql

```diff
@@ -0,0 +1,14 @@
+insert into collection_chapter (
+    title, 
+    document_id, 
+    wordpress_id, 
+    index_in_parent,
+    chapter_path)
+values (
+    $1, 
+    (select id
+    from document
+    where title = $2), 
+    $3, 
+    $4,
+    text2ltree ($5))
```

Comment:
Did you try without `text2ltree` and find that you do actually need it? I didn't think you would need it.

Reply:
yes -- when I remove text2ltree I get this error: expected struct `PgLTree`, found struct `std::string::String`

Reply:
Oh! That means sqlx has a Rust type for ltrees, which I didn't know. Just quickly searched [their docs and found it](https://docs.rs/sqlx/latest/sqlx/postgres/types/struct.PgLTree.html). I would suggest using that type because then we confirm that our string is a valid path before even getting to the DB. Check out [this `from_str` function](https://docs.rs/sqlx/latest/sqlx/postgres/types/struct.PgLTree.html#impl-FromStr) which you should be able to use like this:

```rs
PgLTree::from_str(s)?
```

Reply:
Cool, fixed!